src/visumorph/change_hue.py

```diff
@@ -0,0 +1,32 @@
+def change_hue(image, delta_hue):
+    """
+    Adjusts the hue of the given image.
+
+    This function modifies the hue of each pixel in the input image by the specified amount (delta_hue),
+    changing the overall color composition without altering luminance or saturation.
+
+    Parameters:
+    image (Image Object): The image to be processed. Must be a valid Image Object as defined in the VisuMorph package.
```

Comment:
NumPy format required us to specify the parameters in the form of `name : type` (Refer to the [documentations](https://numpydoc.readthedocs.io/en/latest/format.html#parameters))

By the way, using a customized Image object instead of a numpy array seems like a good idea! But since Marco already wrote a function which accepts a file name and output a numpy array,I think we will need to make **all of our functions consistent** in this regard before submitting the milestone 1.

@atabak-alishiri @Marcony1 @shawnhu444 Please let me know what you think, thanks!

Reply:
I can change my function if you consider that's better guys. I also think that an image object could work

Reply:
Seconded, I think making the whole thing object-oriented is the way to keep it organized

Reply:
@SoloSynth1 Thanks a lot Orix for the formatting notes. we can discuss the input and output format, I am also willing to change my function. @Marcony1

Reply:
You're welcome! I think you can refer to our functions for ideas what the input and output should be.

https://github.com/UBC-MDS/VisuMorph/blob/77706bb3791442c064bbfd006d2147df42cf2145/src/visumorph/rotate.py#L8-L22

https://github.com/UBC-MDS/VisuMorph/blob/77706bb3791442c064bbfd006d2147df42cf2145/src/visumorph/flipping.py#L4-L14

Reply:
Thanks a lot, it should be okay now.

Reply:
Resolving this as commit abea67a3ba0a9fc4f1b9d9dd5f1a0670786f717d addressed this issue